### PR TITLE
Patch 6

### PR DESCRIPTION
**Pull request by rumpli**
next cf release, including now the buildpacks in the release notes

### CODE REVIEW

File: release-notes.html.md.erb [hunk 1]
```diff
@@ -4,6 +4,26 @@ title: APC Release Notes
 
 <strong><%= modified_date %></strong>
 
+## <a id='6.7.0'></a> 6.7.0
+
+Release date: xx.xx.2017
```
Comment (cloudlena) [inline release-notes.html.md.erb]: Please fill in the release date

**Review by cloudlena — CHANGES_REQUESTED** [review]

File: release-notes.html.md.erb (hunk 1 above)
Comment (rumpli) [reply release-notes.html.md.erb]: Will follow as soon the date is known. 👍

Comment (cloudlena) [reply release-notes.html.md.erb]: Ok. Let's wait with merging until then.

**Comment by rumpli** [comment]
Release date is now fixed.

**Review by cloudlena — APPROVED** [review]

**Comment by cloudlena** [comment]
Thanks, @rumpli!

**Pull request merged**